package.json

```
@@ -1,11 +1,12 @@
 {
-  "name": "ArcadeNodeJSSDK",
-  "version": "1.0.0",
+  "name": "ultimate-arcade-server-sdk",
```

let's create npmjs company scope, that should look far more professional and we can put both SDKs below it
```suggestion
  "name": "@ultimatearcade/server-sdk",
```